5.6/docker-entrypoint.sh

```diff
@@ -17,7 +18,9 @@ if [ "$1" = 'mysqld' ]; then
 		fi
 		
 		echo 'Running mysql_install_db ...'
-		mysql_install_db --datadir="$DATADIR"
+		# pass --defaults-file and --defaults-extra-file to mysql_install_db
+		DEFAULTS_FILE_ARGS=$(egrep -o -- '--defaults(|-extra)-file=[^ ]+' <<< "$@")
+		mysql_install_db --datadir="$DATADIR" $DEFAULTS_FILE_ARGS
```

Comment:
This change is a lot more fragile (and thus controversial) than the `SET @@SESSION.SQL_LOG_BIN` is.